packages/browser-wallet/src/popup/pages/Account/i18n/en.ts

```diff
@@ -94,72 +94,72 @@ const t = {
+                body: 'When updating your validator stake you can choose to increase or decrease your stake. If you increase your stake, this is most often effective from the next pay day. If the transaction occurs too close to the next pay day, the update will be effective from the following pay day.\n\nIf you decrease your stake, there is a longer cool-down period. During the cool-down period, the staked amount continues to bake and earn rewards.\n\nYou can also adjust whether you want rewards restaked or not. If the transaction is not made too close to the next pay day, it will take effect then; otherwise it will be effective at the next pay day over.',
             },
             '3': {
                 title: 'Update pool settings',
-                body: 'If you choose to update pool settings, you have the following things to adjust. Changes to the individual parameters are optional, so you don’t have to change them all.\n\n<strong>Pool status</strong>:\n  • Open pool: open a pool for\n     a previously closed baker\n  • Closed for new: close the pool to new delegators.\n     Existing delegators are not affected.\n  • Close pool: close a pool for all delegators.\n\n<strong>Metadata URL</strong>:\n  • The metadata URL can be changed, removed,\n     or left the same.',
+                body: 'If you choose to update pool settings, you have the following things to adjust. Changes to the individual parameters are optional, so you don’t have to change them all.\n\n<strong>Pool status</strong>:\n  • Open pool: open a pool for\n     a previously closed validator\n  • Closed for new: close the pool to new delegators.\n     Existing delegators are not affected.\n  • Close pool: close a pool for all delegators.\n\n<strong>Metadata URL</strong>:\n  • The metadata URL can be changed, removed,\n     or left the same.',
             },
             '4': {
                 title: 'Update pool settings',
-                body: 'There is a cool-down period if you choose to close a pool. During this time, all delegators continue in the pool, and the pool continues to earn rewards if it bakes.\n\nAfter the cool-down, the delegators are removed and the pool will be closed. The baker will keep baking with your own stake afterwards.\n\nChanges to the metadata URL take effect from the next pay day unless the transaction is made too close to the pay day. In that case, it will take effect from the next pay day over.',
+                body: 'There is a cool-down period if you choose to close a pool. During this time, all delegators continue in the pool, and the pool continues to earn rewards if it bakes.\n\nAfter the cool-down, the delegators are removed and the pool will be closed. The validator will keep validation with your own stake afterwards.\n\nChanges to the metadata URL take effect from the next pay day unless the transaction is made too close to the pay day. In that case, it will take effect from the next pay day over.',
```

Comment:
I don't think you can say "The validator will keep validation with your own stake aftewards".

Reply:
Changed it to "The validator will keep validating with your own stake aftewards".